triggers/daml/Daml/Trigger.daml

```diff
@@ -24,6 +25,11 @@ data Identifier = Identifier
   , entityName : Text
   } deriving (Show, Eq)
 
+data AnyContractId = AnyContractId
+  { templateId : Identifier
+  , contractId : Text
```

Comment:
Can we use something like `ContractId ()` here? There as a DAML-LF function `coerce_contract_id : ContractId a -> ContractId b` which we could use then.

Reply:
We could but I think it makes more sense to keep this consistent with `AnyTemplate`. I also deliberately don’t want to use the DAML-LF `ContractId` type to avoid annoyances around the absence of things like `Ord` instances.